bears/general/SpaceConsistencyBear.py

```diff
@@ -1,38 +1,56 @@
-from coalib.bearlib import deprecate_settings
 from coalib.bearlib.spacing.SpacingHelper import SpacingHelper
 from coalib.bears.LocalBear import LocalBear
 from coalib.results.Diff import Diff
 from coalib.results.Result import Result
+from coalib.bearlib.aspects.Formatting.Blankline import NewlineAtEOF
+from coalib.bearlib.aspects.Formatting.Spacing import (
+    Indentation,
+    TrailingSpace
+)
 
 
-class SpaceConsistencyBear(LocalBear):
+
+class SpaceConsistencyBear(
+    LocalBear,
+    aspect={
+        'detect': [
+            Indentation, 
+            NewlineatEOF,
+            TrailingSpace
+        ]
+    },
+    languages=['Python']):
+    
+    @map_setting_to_aspect(
+        enforce_newline_at_EOF = NewlineAtEOF,
+        allow_trailing_spaces = TrailingSpace,
+        indent_size = Indentation,
+        use_spaces = Indentation
```

Comment:
Like above `Indentation.indent_type`. But this line should be dropped, see my comment below.